### ISSUE TITLE
Possible node version issue?

### ISSUE THREAD

**Issue by benjy** [kriskbx/gitlab-time-tracker#5]
```
~ gtt config

/home/vagrant/.nvm/versions/node/v5.11.1/lib/node_modules/gitlab-time-tracker/gtt-report.js:60
let config = new Config(process.cwd());
^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:387:25)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:148:18)
    at node.js:405:3
```

Node version is 5.11.1

**Comment by Felix-IR** [comment]
You need a node version version >= 6 like written in the README

**Comment by kriskbx** [comment]
@Felix-IR is right. I'm sorry but gtt makes heavy usage of basic ES6 which is only supported by Node.js version 6 and higher. You can use the [Node Version Manager](https://github.com/creationix/nvm) to keep your current Node.js version and run gtt with a more recent version. I'm also working on a Docker image that you can safely use. I will keep you up-to-date within this thread when I push changes regarding the Docker image.

**Comment by benjy** [comment]
Great thanks.

**Comment by kriskbx** [comment]
I just released version 1.5.2 with Docker support. You can find the documentation how to use the Docker image over here: https://github.com/kriskbx/gitlab-time-tracker#docker

The image itself is stored on Docker hub: https://hub.docker.com/r/kriskbx/gitlab-time-tracker But you can also clone the repository and build it youself with `docker build --tag "kriskbx/gitlab-time-tracker:latest" .`

If you have any further questions or difficulties with the Docker image feel free to post them here.
